nubis/bin/consul2unixusers.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/python
+
+import os
+import sys
+import pwd
+import ConfigParser
+import optparse
+import consul
```

Comment:
There is an issue open at the moment where we want to install consulate (https://pypi.python.org/pypi/consulate) in nubis-base might want to consider using that library? Although that would require changing up this script